index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Swiggy</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <img src="images/unnamed.png">
+    </header>
+    <section>
+        <header>
+        <img src="https://images.unsplash.com/photo-1625220194771-7ebdea0b70b9?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2070&q=80">
+        <p>Punjabi Momos</p>
+    </header>
+    <article>
+        <div class="navbar">
+            <ul>
+                <li class="heading">Recommended</li>
+                <li>Chandni Chowk Ki Biryani</li>
+                <li>Naughty Momos</li>
+                <li>Naughty Momo Roll</li>
+            </ul> 
+        </div>
+        <div class="menu">
+            <h1>Recommended</h1>
+            <h6>24 ITEMS</h6>
```

Comment:
Why have we used `h6` directly after `h1`? Headings have a semantic meaning - they are not supposed to be used as a replacement of styles (i.e. bigger font does not mean that it would be a larger heading, an h6 can very well look bigger than an h2)

Read Headings section here https://webaim.org/techniques/semanticstructure/#:~:text=A%20heading%20describes%20the%20content,skip%20from%20heading%20to%20heading for an idea